CDCgov/prime-reportstream
# 11113: MSH/SFT to MessageHeader

This PR adds the mappings for MSH/SFT -> MessageHeader -> MSH/SFT

Test Steps:
1. Make sure the tests pass
2. Verify the mappings
  - https://docs.google.com/spreadsheets/d/13pgda5xl-PwCgB9j0axyymwwv7RJVcrIzY8Ah1y1Y1M/edit#gid=0
  - https://docs.google.com/spreadsheets/d/1XsVWY6amPZPo_zDxRywAxAJqX1_BzD8-7gt729NKdH4/edit#gid=0

## Changes
- MSH/SFT -> MessageHeader mapping
- MessageHeader -> MSH/SFT

## Checklist

### Testing

### Process

## Linked Issues
- #11113 

## To Be Done

## Specific Security-related subjects a reviewer should pay specific attention to
